drakrun/drakrun/tools/test-hvm64-example.cfg

```diff
@@ -0,0 +1,15 @@
+name="test-hvm64-example"
+
+vcpus=1
+
+type="hvm"
+builder="hvm" # Legacy for before Xen 4.10
```

Comment:
I don't think we care for Xen <4.10

Reply:
This was copied from XTF and it's a leftover. I will remove this.